virtualization_15.m

```diff
@@ -0,0 +1,19 @@
+//
+//  virtualization_15.m
+//
+#import "virtualization_15.h"
+
+/*!
+ @abstract Set nestedVirtualizationEnabled. The default is false.
+ */
+void setNestedVirtualizationEnabled(void *config, bool nestedVirtualizationEnabled)
+{
+#ifdef INCLUDE_TARGET_OSX_15
+    if (@available(macOS 15, *)) {
```

Comment:
In addition to performing a version check, the property `nestedVirtualizationSupported` can be queried as well. See `Headers/VZGenericPlatformConfiguration.h`

Reply:
The version checks are there for interopability with macos versions older than macOS 15.
`#ifdef INCLUDE_TARGET_OSX_15` covers builds with a SDK not having the macOS 15 APIs, or if the macOS 15 APIs were explicitly disabled at build time

 `if (@available(macOS 15, *))` is used at runtime, to produce a binary which can use the new API when run on macOS 15, but which can still be run on macOS 14 and older with an appropriate fallback.

My understanding is that `nestedVirtualizationSupported` is useful at runtime when running on macOS 15, for example to know that nested virtualization is not available on an intel mac.

Reply:
I think it is harmless and will simply be a no-op if nested virtualization is not supported.

Nonetheless, the PR has been modified to perform the check before setting the value and it also works fine from my tests.

Thanks for the suggestion @asauber.

Reply:
As @cfergeau said, I think it should really be handled. I'll allow it for now and deal with it in a separate PR.